app/browser/tabs.js

```diff
@@ -798,7 +798,7 @@ const api = {
 
   goToIndex: (tabId, index) => {
     const tab = getWebContents(tabId)
-    if (tab && !tab.isDestroyed() && tab.canGoToIndex(index)) {
+    if (tab && !tab.isDestroyed() && tab.goToIndex(index)) {
```

Comment:
we don't want to call `goToIndex` twice, but we should check that index >= 0 and < the number of entries